### PR TITLE
add multi-line genre editing

### PR DESCRIPTION
**Pull request by gaycodegal**
Adds new string and updates album and song tag editors to allow the one-per-line style input. Genre parsing already handles this, so this was only a UI change

### CODE REVIEW

**Comment by gaycodegal** [comment]
fixes #914

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -61,7 +61,9 @@
     <string name="disc_number">"Disc Number"</string>
     <string name="lyrics">Lyrics</string>
     <string name="album_artist_multiple">Album Artists (one per line)</string>
+    <string name="album_genre_multiple">Album Genres (one per line)</string>
```
Comment (soncaokim) [inline app/src/main/res/values/strings.xml:64]: We can drop this, album genre and (track) genre are the same, they all being saving in the same tag.

**Review by soncaokim — CHANGES_REQUESTED** [review]
Can you please drop the `album_genre_multiple` and use 'genre_multipe` instead string?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (gaycodegal) [reply app/src/main/res/values/strings.xml]: Done

**Comment by gaycodegal** [comment]
Now I do agree with you but if you look at the artist line, the same logic could apply there. If you look up the artist id3 tags they should be being set per song and not on a album wide basis, unless we're doing something extra spicy behind the scenes (I didn't check, just assuming). So would you like me to also remove the album_artist_multiple and just use artist_multiple?

**Comment by soncaokim** [comment]
Hi,

There are two separated tags for per-song and per-album artist(s).

Thanks for the modifications on the genre.

S

**Review by soncaokim — APPROVED** [review]
Thanks

**Pull request merged**